app/models/compute_resources/foreman/model/ec2.rb

```diff
@@ -12,6 +14,19 @@ def to_label
       "#{name} (#{region}-#{provider_friendly_name})"
     end
 
+    def gov_cloud=(enable_gov_cloud)
+      if enable_gov_cloud == '1'
```

Comment:
Is a checkbox always normalized to `'1'` or `'0'` or should it handle booleans?

Reply:
I believe it is, from API people would use region directly.

Reply:
This should work fine, rails takes care of this. You basically get a hidden field with `0` and a checkbox that overrides that with `1` if checked.